.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: android
 env:
-    - BUILD_TYPE=android TERM=dumb
```

Comment:
I was wondering why the tests are not shown correctly in travis, like the name of each one, and this seems to be the reason. The logs in the console would be clearer without this, in case it can be removed.